Behavioral.Automation.Selenium/Behavioral.Automation/Services/ConfigServiceBase.cs

```diff
@@ -29,7 +29,11 @@ public static class ConfigServiceBase
         private const string AccessClipboardString = "ACCESS_CLIPBOARD";
         
         private const string BrowserBinaryLocationString = "BROWSER_BINARY_LOCATION";
+        
+        private const string UnhandledPromptBehaviorString = "UNHANDLED_PROMPT_BEHAVIOR";
```

Comment:
Let's update the readme and config file?

Reply:
config file updated